api/src/services/listing.service.ts

```diff
@@ -1531,7 +1539,7 @@ export class ListingService implements OnModuleInit {
   */
   async process(): Promise<SuccessDTO> {
```

Comment:
I know this isn't part of your change, but now that we have two cron jobs in this file can this function be renamed to be specific to the listing cron job

Reply:
What should I name the function? `listingCronJob` or `closeListings`

Reply:
I think having the word "close" in it would useful